ansible/configs/sap-integration/files/fio-test-job.yaml.j2

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
```

Comment:
I doubt you are using this since I created it for a special use in another config. Please ensure that all the files you are using are needed in this config.

If you are using it, you probably shouldn't be :)

Reply:
done